circuit/types/group/src/helpers/mod.rs

```diff
@@ -17,6 +17,7 @@
 use super::*;
 
 pub mod from_bits;
+pub mod from_unchecked;
```

Comment:
If we're going to feature gate, we should just put `#[cfg(feature = "unchecked")]` on this module directly.

Reply:
I've removed the "unchecked" feature flag.